### ISSUE TITLE
onnx parsing fails

### ISSUE THREAD

**Issue by faculty-hieonn** [levihsu/OOTDiffusion#183]
Hi, thanks for your stunning works. 

I've tried to use the artifacts you provided by the command 
python run_ootd.py --gpu_id 0 --model_path ../data/model_5.png --cloth_path ../data/00151_00.jpg --scale 2 --sample 4

but I've met the error message like following
/onnxruntime/capi/onnxruntime_inference_collection.py", line 452, in _create_inference_session
    sess = C.InferenceSession(session_options, self._model_path, True, self._read_config_from_model)
onnxruntime.capi.onnxruntime_pybind11_state.InvalidProtobuf: [ONNXRuntimeError] : 7 : INVALID_PROTOBUF : Load model from /home/..../OOTDiffusion/checkpoints/humanparsing/parsing_atr.onnx failed:Protobuf parsing failed.

I've installed through requirements.txt so that onnxruntime 1.16.2 has been installed. 

I have no clue to resolve it for now, I want you to help it so that other people can experience your works. 

Thanks.

**Comment by levihsu** [comment]
Hi, please download the model checkpoints from https://huggingface.co/levihsu/OOTDiffusion/tree/main/checkpoints

**Comment by faculty-hieonn** [comment]
> Hi, please download the model checkpoints from https://huggingface.co/levihsu/OOTDiffusion/tree/main/checkpoints

Dear, Thanks for reply and I've forgotten mentioning the exact status. I've downloaded the checkpoints when I had tested your work. 3 folders.  

The main reason why I've asked you is that the error message says the checkpoint would be corrupted. I don't think it would be corrupted but I think I have missed somthing in environment or something.

Thanks!

**Comment by levihsu** [comment]
I see. It seems to be an environmental error and the onnx file is not successfully loaded. Could you check your environment and files again?

**Comment by faculty-hieonn** [comment]
> I see. It seems to be an environmental error and the onnx file is not successfully loaded. Could you check your environment and files again?

Yes, so I want to have more clues to check out. It there any check point for the environmetal issues? 

Thanks!

**Comment by faculty-hieonn** [comment]
Reopened for the tracking!

**Comment by rohit22059** [comment]
@faculty-hieonn I'm facing the same issue. Did you resolve the error? Could you please guide how can I resolve for the same?
Thanks
